gammapy/estimators/points/sensitivity.py

```diff
@@ -135,6 +141,11 @@ def run(self, dataset):
         criterion = self._get_criterion(
             excess.data.squeeze(), dataset.background.data.squeeze()
         )
+        logging.warning(
```

Comment:
Does this require a deprecation warning? It is a change in the output format, not in the API.

Reply:
Thanks @registerrier ! Interpreting backwards compatibility loosely, I tend to see it as "something which breaks my script which used to work on a prev version". In this case, users might like to be fore-warned that `table["energy"]` would yield errors in future versions?